backend/clubs/serializers.py

```diff
@@ -1409,10 +1409,25 @@ class UserClubVisitSerializer(serializers.ModelSerializer):
 
     person = serializers.HiddenField(default=serializers.CurrentUserDefault())
     club = ClubListSerializer(read_only=True)
+    ip = serializers.HiddenField(default=None)
+
+    def save(self):
+        if not self.instance:
+            if not self.validated_data.get("ip"):
```

Comment:
You can remove this if statement, you don't want to save the IP if the user specifies it (always trust the server, no reason for user to specify it).

Reply:
Ok will do!